src/components/SharedComponents/Map/DetailsMap.tsx

```diff
@@ -13,35 +11,38 @@ import {
 } from "components/SharedComponents";
 import { View } from "components/styledComponents";
 import { t } from "i18next";
-import type { Node } from "react";
 import React, { useState } from "react";
+import { GestureResponderEvent } from "react-native";
+import { Region } from "react-native-maps";
 import openMap from "react-native-open-maps";
 import { useSafeAreaInsets } from "react-native-safe-area-context";
+import Observation from "realmModels/Observation";
+import type { RealmObservation } from "realmModels/types";
 import { getShadow } from "styles/global";
 import colors from "styles/tailwindColors";
 
-type Props = {
-  closeModal: Function,
+interface Props {
+  closeModal: () => void,
   coordinateString?: string,
-  headerTitle?: Object,
-  observation?: {
-    latitude?: number,
-    privateLatitude?: number,
-    longitude?: number,
-    privateLongitude?: number,
-    obscured?: boolean
-  },
-  region?: Object,
-  showLocationIndicator: boolean,
-  tileMapParams: Object,
+  headerTitle?: React.ReactNode,
+  observation?: Observation & RealmObservation,
```

Comment:
```suggestion
  // TODO MOB-1038: reconcile the type issues here requiring the intersection
  observation?: Observation & RealmObservation,
```

Discussed this w/ team and we decided to defer a type reconciliation, I'm just adding a comment here to reference that